platform/brains2/src/duo/main.cpp

```diff
@@ -72,6 +72,9 @@ const int led_order[NUM_LEDS] = {1,  2,  3,  4,  5,  6,  7,  8,  9,  10,
 #include "duo-firmware/src/DrumSynth.h"
 #include "drums.h"
 
+// TODO: make this not global. Needed for dac1.stop()
+BoardAudioOutput dac1; // xy=988.1000061035156,100
```

Comment:
Having it here makes sense to me. It's still local to this translation unit, so not really global.